tests/python/gaia-ui-tests/gaiatest/gaia_graphics_test.py

```diff
@@ -48,17 +50,20 @@ def tearDown(self):
         """At the end of test execution, it checks for the errors"""
         self.assertTrue(self.test_passed, msg=self.failcomment)
 
-    def take_screenshot(self):
-        """invokes screen capture event, crops the status bar, and saves to the file"""
+    # hook to override default locale setting
+    # see https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/gaia_test.py#L1019
+    def modify_settings(self, settings):
+        settings['language.current'] = self.locale
```

Comment:
yeah, thought it might also be useful in main gaiatest, but I wanted to see what other people think about it.  If we do agree to put this in gaiatest, yes then we should set it up initially.